waspc/ChangeLog.md

```diff
@@ -903,7 +903,7 @@ This was the file tree of a newly generated project in the previous version of W
 │   ├── Main.css
 │   ├── MainPage.js
 │   ├── .waspignore
-│   └── waspLogo.png
+│   └── waspLogo.svg
```

Comment:
This shouldn't be changed, this and the line below, since this is ChangeLog -> so this is how it was at that point in the time and should be kept that way.